src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -36,12 +37,55 @@ protected override Task<ImmutableArray<ISymbol>> GetSymbolsAsync(SyntaxContext c
                 context.SemanticModel, position, context.Workspace, options, cancellationToken);
         }
 
+        protected async override Task<bool> ShouldProvidePreselectedItemsAsync(CompletionContext completionContext, SyntaxContext syntaxContext, Document document, int position, Lazy<ImmutableArray<ITypeSymbol>> inferredTypes, OptionSet options)
+        {
+            var sourceText = await document.GetTextAsync(CancellationToken.None).ConfigureAwait(false);
+            if (ShouldTriggerInArgumentLists(sourceText, options))
+            {
+                // Avoid preselection & hard selection when triggered via insertion in an argument list.
```

Comment:
can you clarify why for the next person who maintains this?  i.e. give an example of why this would be bad.

Reply:
done